build.sh

```diff
@@ -391,6 +384,11 @@ perl -pe "s|<!--SCRIPTS-->|${scripts:8:$((${#scripts}-10))}|s" "$SRCDIR/safari/I
 | perl -000 -p -e 's|(<key>(?:CFBundleShortVersionString\|CFBundleVersion)</key>\s*)<string>[^<]*</string>|$1<string>'"$VERSION"'</string>|sg' \
 > "$BUILD_DIR/safari.safariextension/Info.plist"
 
+# Transpile Safari JS for Safari 10.0<
+echo "Transpiling Safari JS..." >> "$LOG";
+./node_modules/babel-cli/bin/babel.js "$BUILD_DIR/safari.safariextension/" --out-dir "$BUILD_DIR/safari.safariextension/" --presets es2015 -q >> "$LOG" 2>&1
```

Comment:
Here and elsewhere, should be an absolute path using `$CWD` (which is badly named, but we can fix that another time). No guarantee that the script will be run from the root.